packages/forma-36-react-components/src/components/Icon/Icon.tsx

```diff
@@ -281,20 +281,17 @@ const iconComponents = {
 };
 
 export type IconType = keyof typeof iconName;
-export type IconColorType =
```

Comment:
Weren't these types used somewhere?

Reply:
I searched it in vscode and didn't find any use
Running `tsc` does not fail either